pkg/agent/flowexporter/connections/conntrack_connections_perf_test.go

```diff
@@ -0,0 +1,180 @@
+// +build !race
+
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"crypto/rand"
+	"flag"
+	"fmt"
+	"math/big"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/agent/flowexporter"
+	connectionstest "antrea.io/antrea/pkg/agent/flowexporter/connections/testing"
+	"antrea.io/antrea/pkg/agent/flowexporter/flowrecords"
+	interfacestoretest "antrea.io/antrea/pkg/agent/interfacestore/testing"
+	"antrea.io/antrea/pkg/agent/openflow"
+	proxytest "antrea.io/antrea/pkg/agent/proxy/testing"
+	queriertest "antrea.io/antrea/pkg/querier/testing"
+	k8sproxy "antrea.io/antrea/third_party/proxy"
+)
+
+const (
+	testNumOfConns        = 100000
+	testNumOfNewConns     = 10000
+	testNumOfDeletedConns = 10000
+)
+
+var (
+	connStore      *ConntrackConnectionStore
+	conns          []*flowexporter.Connection
+	mockConnDumper *connectionstest.MockConnTrackDumper
+)
```

Comment:
If possible, avoid global variables here. Wrap these in a struct and populate / return the struct during test setup.